src/DocumentFormat.OpenXml/Packaging/OpenXmlPart.cs

```diff
@@ -480,17 +443,154 @@ public void ValidateXml(string schemaFile, ValidationEventHandler validationEven
         /// Gets the root element of the current part.
         /// Returns null when the current part is empty or is not an XML content type.
         /// </summary>
-        public OpenXmlPartRootElement? RootElement
+        public OpenXmlPartRootElement? RootElement => PartRootElement;
+
+#endregion
+
+#region Linq to XML
+
+        /// <summary>
+        /// Gets or sets
+        /// </summary>
+        public XElement? RootXElement
```

Comment:
Should this be exposed via the `OpenXmlPartRootElement` instead? That way it would always be in sync

Reply:
It should not be exposed that way, because the `OpenXmlPartRootElement` only plays a role if both the Linq to XML approach and the strongly typed classes are used at the same time.

Reply:
I'm fine with that